vars/build.groovy

```diff
@@ -1,13 +1,11 @@
 #!/usr/bin/groovy
 import io.fabric8.Events
```

Comment:
@piyush1594 could you move Events to io.openshift. as well please?

Reply:
ok